components/scream/src/share/atm_process/atmosphere_process_group.cpp

```diff
@@ -209,6 +215,12 @@ void AtmosphereProcessGroup::run_sequential (const Real dt) {
     atm_proc->set_update_time_stamps(do_update);
     // Run the process
     atm_proc->run(dt);
+#ifdef SCREAM_HAS_MEMORY_USAGE
+    long long my_mem_usage = get_mem_usage(MB);
+    long long max_mem_usage;
+    m_comm.all_reduce(&my_mem_usage,&max_mem_usage,1,MPI_MAX);
+    m_atm_logger->info("[EAMxx::run_sequential::"+atm_proc->name()+"] memory usage: " + std::to_string(max_mem_usage) + "MB");
```

Comment:
Imho, reporting mem usage of all atm procs during each timestep is more of a "debug" log level than an "info" log level.